docs/organizer-guide/meetups/faq-meetups.rst

```diff
@@ -0,0 +1,34 @@
+Meetups FAQ
+===========
+
+The following list of questions and answers cover some issues that you might
+encounter when you want to `start a meetup <http://www.writethedocs.org/organizer-guide/meetups/starting/>`_.
+
+Can Write the Docs cover the Meetup.com fees?
+---------------------------------------------
+
+Yes! We can cover the website fees for active meetups, or for new meetups on a trial-period basis.
+
+**New meetup groups:** If you want to start a new meetup group but need assistance with the fees,
+please `contact us <mailto:conf@writethedocs.org>`_ **before** you create the meetup group, so that
+we can create the group directly from the Write the Docs organization account. The trial period lasts
+for 3 months, which should give you enough time to establish a local presence and hopefully run a few meetups.
```

Comment:
We should probably be more explicit about how we evaluate groups are the trial period.